deploy/bin/build

```diff
@@ -46,8 +46,16 @@ cmd_app() {
     --exclude='./worker-types-backup.json' \
     --exclude='./docker-worker-amis.json' \
     --exclude='./packer-artifacts.json' \
+    --exclude='docker-worker.tar.gz' \
+    --exclude='modules.tar.gz' \
+    --exclude='kernel.tar.gz' \
+    --exclude='initrd.tar.gz' \
```

Comment:
Would it be better to whitelist the subdirectories required here?  I've been bitten multiple times by things that tar up a repo when I have some temporary file there containing secrets..

Reply:
Yep, I can do that in a follow up PR